README.md

```diff
@@ -4,7 +4,7 @@ ncdfchecker.py is a config driven utility that will allow you to check that a
 NetCDF4 file is compliant with a standard specified in a particular config, as
 provided in a json file.
 
-usage: ncdfchecker.py [-h] [--quiet] [--strict] input_file config_path
+usage: ncdfchecker.py [-h] [--quiet] [--strict] [--initdate INITDATE] [--runlength RUNLENGTH] input_file config_path
```

Comment:
If you are providing command line overrides there needs to be an equivalent config entry - the tool is first and foremost config file driven.

Runlength is not suitable in a generic case - again, in the spirit of the tool rather than the specific application you're planning on using it for. min_date and max_date would be in line with naming conventions and structure elsewhere.

Reply:
As discussed offline, the changes now only use the "forecast_reference_time" in the NetCDF meta-data to convert the leadtimes in the file to datetime objects so that we can grab the required period. I have therefore removed these extra arguments from the checker script.